### PR TITLE
#45 UI/UX About us and Contact Page/about

### PR DESCRIPTION
**Pull request by MinhHoaNguyen**
The Contact Us form was added to the About page

### CODE REVIEW

File: app/about/page.tsx [hunk 1]
```diff
@@ -6,86 +6,146 @@ const DEFAULT_IMAGE = "/icons/app/stellarHorizons.png" as const;
...
+    image: "/images/team/Ryan.png",
+    description: "Backend Developer focused on stripe and prisma integration.",
+    email: "",
   },
 ];
 
 const AboutPage = () => {
   return (
     <div className="min-h-screen bg-gray-100 text-gray-800 p-8">
-      {/* Team Introduction */}
```
Comment (Luceium) [inline app/about/page.tsx:59]: This seems like a good idea to keep. I'm not sure why we removed it.

File: app/about/page.tsx [hunk 2]
```diff
@@ -6,86 +6,146 @@ const DEFAULT_IMAGE = "/icons/app/stellarHorizons.png" as const;
...
-      <section className="text-center mb-12">
-        <h1 className="text-5xl font-bold mb-6">Meet the LikeHome Team</h1>
-        <p className="text-lg">
-          We are a group of passionate developers committed to building the best
-          hotel-booking platform.
-        </p>
-      </section>
+      {/* Main Flex Container */}
+      <div className="flex flex-col lg:flex-row gap-8">
```
Comment (Luceium) [inline app/about/page.tsx:68]:
```suggestion
      <div className="flex flex-col gap-8">
```
Contact us is typically at the bottom. Also this layout has a lot of dead space.

File: app/about/page.tsx [hunk 3]
```diff
@@ -6,86 +6,146 @@ const DEFAULT_IMAGE = "/icons/app/stellarHorizons.png" as const;
...
+        {/* Contact Us Form */}
+        <div className="bg-white p-6 rounded-lg shadow-lg flex-[1] lg:flex-[0.4]">
+          <h2 className="text-3xl font-bold mb-4 text-center">Contact Us!</h2>
```
Comment (Luceium) [inline app/about/page.tsx:71]: Components help readability. Contact us could and should be in a component.

File: app/about/page.tsx [hunk 4]
```diff
@@ -6,86 +6,146 @@ const DEFAULT_IMAGE = "/icons/app/stellarHorizons.png" as const;
...
+                quality={100}
               />
-              <h3 className="text-2xl font-semibold mb-2">{member.name}</h3>
-              <p>{member.description}</p>
-            </div>
-          ))}
+                <h3 className="text-xl font-semibold mb-2">{member.name}</h3>
+                <p>{member.description}</p>
+                <p className="text-sm text-gray-600">
+                  <a href={`mailto:${member.email}`} className="hover:text-blue-500">
```
Comment (Luceium) [inline app/about/page.tsx:140]: I think I linkedin link would be make more sense. I wouldn't want to give out my email.

**Review by Luceium — CHANGES_REQUESTED** [review]

**Review by Luceium — APPROVED** [review]
lgtm

**Review by luSteven01 — APPROVED** [review]
looks good to me

**Pull request merged**